packages/backend/src/shared/pagination.test.ts

```diff
@@ -333,6 +341,65 @@ describe('Pagination', (): void => {
           }
         )
       })
+      describe('webhookEvents', (): void => {
```

Comment:
I'm not sure if someone else would agree with me, but I don't see the need to test `getPageInfo` for all of the resources. It's outside the scope of this test to test everything that `getPage` _could_ be, all this function knows/should know about is the type definition. As long as we give it a proper function to test, I think we should reduce this test suite.

As long as we test `getPage` functions in each of the service, (as you do here: https://github.com/interledger/rafiki/pull/1454/files#diff-f6ae8e955305efe9c101fc4a66e9c257713a099b0018dcc5061a08983b854134R99), we are good to remove this IMO

Reply:
I happened to find these tests by accident - very easily could have missed them. My initial reaction was kinda similar in that I didn't think these tests should concern themselves with every resource that uses them. It seems analogous to `getPageTests` so I thought (ideally) there should be something like `getPageInfoTests` that are exported and run in whichever resource uses them. But I suppose I can also see an argument for testing `getPageInfo` once and then just ensuring each resource calls it correctly (side note, but wouldn't this also apply to `getPageTests`? does being a method on the inherited `baseModel` instead of a `utility` function make some difference there?).

In fact that is really what I am interested in with `getPageInfo` is ensuring the right function is passed in (it needs the `pagination` from the `getPageInfo` scope but still needs the filter received by the resolver, which should also be the filter passed into the service's `getPage`). But this doesn't test that.

> As long as we test getPage functions in each of the service

I think it's actually the resolver tests that need it (that's where it's called - service doesn't know about it) but I think I agree in principle.